exercises/custom-set/test.ml

```diff
@@ -1,39 +1,39 @@
 open OUnit2
 
 module type EXPECTED = sig
-    type t
-    val empty : t
-    val equal : t -> t -> bool
-    val is_empty : t -> bool
-    val is_member : t -> int -> bool
-    val to_string : t -> string
-    val of_list : int list -> t
-    val to_list : t -> int list
-    val add : t -> int -> t
-    val remove : t -> int -> t
-    val difference : t -> t -> t
-    val intersect : t -> t -> t
-    val union : t -> t -> t
+  type t
+  val empty : t
+  val equal : t -> t -> bool
+  val is_empty : t -> bool
+  val is_member : t -> int -> bool
+  val to_string : t -> string
+  val of_list : int list -> t
+  val to_list : t -> int list
+  val add : t -> int -> t
+  val remove : t -> int -> t
+  val difference : t -> t -> t
+  val intersect : t -> t -> t
+  val union : t -> t -> t
 end
 
 module CSet : EXPECTED = Custom_set.Make(struct
-    type t = int
-    let compare a b = compare (a mod 10) (b mod 10)
-    let equal a b = compare a b == 0
-    let to_string v = string_of_int v
+  type t = int
+  let compare a b = compare (a mod 10) (b mod 10)
+  let equal a b = compare a b = 0
+  let to_string v = string_of_int v
 end)
 
 let rec int_list_eq l1 l2 = match (l1, l2) with
-    | (h1::t1), (h2::t2) -> h1 == h2 && int_list_eq t1 t2
-    | [], [] -> true
-    | _, _ -> false
+  | (h1::t1), (h2::t2) -> h1 == h2 && int_list_eq t1 t2
```

Comment:
I'm not the original author - my change was only in indentation. I've fixed this use of ```==```, and another in the sample implementation.